examples/nlp/joint_intent_slot_infer_b1.py

```diff
@@ -2,7 +2,6 @@
 
 import numpy as np
 from pytorch_transformers import BertTokenizer
```

Comment:
i think we've decided to move to transformers?

Reply:
I think we decided to wait with transformers since it breaks Bert pretraining

Reply:
I am assuming this is resolved